tests/unit/test_ops.py

```diff
@@ -640,16 +657,16 @@ def test_difference_lag():
     processor.fit(dataset)
     new_gdf = processor.transform(dataset).to_ddf().compute()
 
-    assert new_gdf["timestamp_difference_lag_1"][0] is None
+    assert new_gdf["timestamp_difference_lag_1"][0] is cudf.NA
```

Comment:
Perhaps you can add a `hasattr(cudf, "NA")` check for backward cudf compatibility?

Reply:
That makes sense. Added. Thanks Rick!